eng/pipelines/notifications.yml

```diff
@@ -76,23 +77,13 @@ stages:
             --organization $(Organization) `
             --project $(Project) `
             --path-prefix "\$(PathPrefix)" `
-            --dev-ops-token-var DEVOPS_TOKEN `
```

Comment:
After this change gets merged lets be sure to clean-up and remove these secrets from the variable group we are consuming for this pipeline.

Reply:
Put it in issue: https://github.com/Azure/azure-sdk-tools/issues/2745